### ISSUE TITLE
How will we allow users to select a certain project to donate to?

### ISSUE THREAD

**Issue by coderbyheart** [distributeaid/distributeaid.org#785]
We show donations per project, but none of the donations options allow to select a project.

## Solution

- [x] Change wording around donation progress: projects have a need amount in EUR.
- [x] Add fund allocation ledger per project: date, amount (in EUR), description
- [ ] ~~Progressbar becomes percentage allocated for project from total donated amount~~ Most projects have unlimited funding needs (at least when it comes to the funds DA can raise)
- [x] Add total donated amount (editable on donation page), in EUR

**Comment by jtfairbank** [comment]
@coderbyheart We can create different projects on Open Collective, or ask donors to specify the project on the memo line of a PayPal / bank transfer / check transaction.  I believe Ellie & Florence were also looking into Donor Box which supports project funding models and can be embedded directly into our site.

We do want to be careful about turning our projects / responses into a popularity contest when it comes to funding.  This page should connect donors to one of the impacts that Distribute Aid makes, but the donation is not necessarily limited to that project / response.  It's ok if folks make general / unallocated donations even if they make it through a specific project, and I'll talk with Ellie & Florence to see if we need to put up specific language clarifying this.

**Comment by coderbyheart** [comment]
I agree. I'd prefer that we have one donation pot, and DA can allocate funds as we see fit. We can still communicate funding **needs** per project, but only a number, without progress.

**Comment by coderbyheart** [comment]
Status:

> great question, we did not think about this. Do you have any suggestions, we will also brainstorm with Taylor and let you know.

**Comment by jtfairbank** [comment]
@coderbyheart what if we provide pathways for folks to donate to the general fund thru projects, but then track the "progress" of the project's funding by how much we've been able to budget to the project from the general fund?

That way each project can have goals + show progress towards those goals, but that's getting filtered through our team so we can prioritize the budgets and balance them out across projects?

Basically: Donor => Project => Donate to General Fund => Budgeting Decision => Update Project Budget

**Comment by coderbyheart** [comment]
I think that would be great in terms of transparency! We would then create a simple ledger per project, which lists these budgeting needs and allocations. The overall need is then the sum of all project needs.

**Comment by jtfairbank** [comment]
That sounds perfect!  And it'll be easier to update budget allocations on a regular cycle to show progress than to try and keep up with each / every random donation to show fundraising progress.

**Comment by coderbyheart** [comment]
ACK from project team:
> Lets go with option 2, we also cannot guarantee that the donations will go toward specific projects (it was more to show our donors the value of their donation).

Option 2 was: "remove the individual funds distribution (can we, and do we want to, actually guarantee that donations or allocated ONLY to a specific project, if users chose so)"

**Comment by coderbyheart** [comment]
Updated solution.
